### ISSUE TITLE
documenting multiple R packages in one rtd site

### ISSUE THREAD

**Issue by erikriverson** [ropenscilabs/r2readthedocs#14]
Hello! We are evaluating using this project, our use case is to host documentation for multiple R projects in a single RTD site. The homepage states 

> This package may easily be used to document several packages

I am wondering if there are any examples of this? 

Thank you!

**Comment by mpadge** [comment]
Yes indeed @erikriverson, this whole package was initially developed for [this multiiple-package site](https://ropensci-review-tools.readthedocs.io/en/latest/), via source code [here](https://github.com/ropensci-review-tools/ropensci-review-tools). That's a system of five main inter-dependent packages, plus docs which cross between packages and don't neatly fit into any single package. The whole thing is coordinated by [this relatively simply makefile](https://github.com/ropensci-review-tools/ropensci-review-tools/blob/main/makefile) which in turn mostly relies on [this script](https://github.com/ropensci-review-tools/ropensci-review-tools/blob/main/pkgdocs-script.R).

A lot of that script is general, and likely sufficiently generalisable that it could and should be internalised in package functions here, but we first need some other examples of usage to know how generally appilcable that workflow is. It'd be great in that context if you were to start using this package, so you could give feedback on how to improve it and make it more generally usable. Hopefully you do decide to use it, in which case we'll be happy to work with you to adapt to your specific needs :+1:
